uat/src/main/java/com.aws.greengrass/FileSteps.java

```diff
@@ -118,6 +128,24 @@ private void addDataToFile(String data, Path filePath) throws IOException {
     /**
      * Arranges some log files with content on the /logs folder for a component
      * to simulate a devices where logs have already bee written.
+     *
+     * @param componentName name of the component.
+     * @param nfiles        number of log files to write.
+     */
+    @Then("I verify that {int} temporary rotated log files for component {word} are still available")
```

Comment:
I know this name is copy paste from EF, but it is a bit misleading when we call `getComponentLogFiles` we are including the active file in there. Maybe a better name is

```
I verify that {int} log files for component {word} are still available
```

Reply:
resolved